lighthouse/multi-cluster-service-selection.md

```diff
@@ -0,0 +1,71 @@
+# Multi-Cluster Service Selection Optimization - EP
+
+## Motivation
+
+While the world is moving towards micro-services application based architecture, more organizations combine multi-cluster and multi-cloud providers for redundancy, performance, management, and other reasons.
+We would like to optimize the total cost of a Kubernetes deployment in means of communication between multi-clusters, given that we could decide between different approaches. The three approaches possible, currently, are optimizing the full service chain of a single app/deployment, multiple apps/deployments, or a smart local optimization per cluster point of view, without building the full service chain.
+In this Enhancement Proposal, we shall introduce questions, ideas, and requirements to optimize the service selection mechanism of the Lighthouse Plugin.
+
+
+### Requirements
+1. Topology:
+    1. Selection Decision should be made locally, i.e per cluster - decentralized solution.
+    2. The solution must be adaptive for topology changes.
+1. Cost Parameters/Metrics:

```

Comment:
0. Tunnel status (we may want to implement something like BFD on the gateways, on top of IPSEC/cabledrivers. In such way we get a very early signal (seconds) that something is going wrong with the connectivity. We could revive [1]

[1] https://github.com/jthurman42/go-bfd